### PR TITLE
[Accordion] Update docs

### PR DESCRIPTION
**Pull request by benoitgrelard**

### CODE REVIEW

File: data/primitives/components/accordion/0.0.16.mdx [hunk 1]
```diff
@@ -148,8 +148,8 @@ Contains all the parts of an accordion.
       typeSimple: 'enum',
       description: (
         <span>
-          Determines whether a single panel can be open at a time or
-          multiple panels at the same time.
+          Determines whether a single item or multiple items can be
+          opened at the same time.
```
Comment (andy-hook) [inline data/primitives/components/accordion/0.0.16.mdx:152]:
```suggestion
          Determines whether single or multiple items can be
          opened at the same time.
```

And similar here perhaps?

**Review by andy-hook — APPROVED** [review]

File: data/primitives/components/accordion/0.0.16.mdx [hunk 2]
```diff
@@ -17,7 +17,7 @@ aria: https://www.w3.org/TR/wai-aria-practices-1.2/#accordion
 <Highlights
   features={[
     'Full keyboard navigation.',
-    'Can expand only one panel at a time or multiple.',
+    'Can expand only one or multiple items.',
```
[the remark this answers is not in the document]
Comment (benoitgrelard) [reply data/primitives/components/accordion/0.0.16.mdx:20]: I think perhaps

```suggestion
    'Can expand one or multiple items.',
```

is even clearer

File: data/primitives/components/accordion/0.0.16.mdx (hunk 1 above)
Comment (benoitgrelard) [reply data/primitives/components/accordion/0.0.16.mdx:152]:
```suggestion
          Determines whether one or multiple items can be
          opened at the same time.
```

?

File: data/primitives/components/accordion/0.0.16.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (andy-hook) [reply data/primitives/components/accordion/0.0.16.mdx:20]: Even better 👍

File: data/primitives/components/accordion/0.0.16.mdx (hunk 1 above)
Comment (andy-hook) [reply data/primitives/components/accordion/0.0.16.mdx:152]: 👍

**Pull request merged**